src/legacy/core_plugins/vis_type_table/public/shim/legacy_dependencies_plugin.ts

```diff
@@ -18,12 +18,11 @@
  */
 
 import { CoreStart, Plugin } from '../../../../../core/public';
-import { initTableVisLegacyModule } from './table_vis_legacy_module';
 
 /** @internal */
 export class LegacyDependenciesPlugin implements Plugin {
```

Comment:
I'm not sure whether this file can be removed.

Reply:
I think,  since it doesn't contain any functionality it should be removed.

Reply:
Done.